src/Public/js/myradio.core.js

```diff
@@ -49,7 +49,10 @@ var myradio = {
     );
     return reportButton;
   },
-  callAPI: function (method, module, action, id, firstParam, options, successFunc = function(){}) {
+  callAPI: function (method, module, action, id, firstParam, options, successFunc) {
+    if (successFunc == undefined) {
```

Comment:
"typeof successFunc === 'undefined'" would be better - technically, this will throw an error, e.g.:
```
chicken == undefined
VM1569:1 Uncaught ReferenceError: chicken is not defined
    at <anonymous>:1:1
```